app/Website/database/migrations/2024_03_16_174823_create_blog_posts_table.php

```diff
@@ -20,6 +20,7 @@ public function up(): void
             $table->unsignedBigInteger('author_id');
             $table->date('publish_date')->nullable();
             $table->timestamps();
+            $table->softDeletes();
```

Comment:
This would have to be added in a separate migration, this migration was already executed.